### ISSUE TITLE
Objects365 pretrained model 

### ISSUE THREAD

**Issue by JorgeBorras18** [Deci-AI/super-gradients#1485]
### 💡 Your Question

Just like the title says:
Can anyone provide a Yolo-Nas pretrained model on the objects365 classes?

**Comment by BloodAxe** [comment]
We are not planning to release Objects 365 pretrained weights since we didn't aim to achieve SOTA performance on that dataset. The goal of the pretraining on Objects 365 was pretraining :) and not best possible performance.

**Comment by lixinye666** [comment]
hello , have u finished Objects 365 pre-training model
